odkx-src/build-scripts.rst

```diff
@@ -148,7 +148,27 @@ To run the UI tests:
 
       ./gradlew clean testSnapshotBasicDebug connectedSnapshotBasicDebugAndroidTest
 
+Build Variants
+~~~~~~~~~~~~~~~~~~
+
+you can configure build variants to create different versions of your app from a single project, and how to properly manage your dependencies and signing configurations. Each build variant represents a different version of your app that You can configure a dependency for a specific build variant. For more information refer to https://developer.android.com/studio/build/build-variants
```

Comment:
I am confused by what you mean by different versions. While it's a true statement a version number does not correspond to a build variant (it could, but it does not). You can also probably remove signing configurations as people do not need to worry about that.